lib/new_utils.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * @param {String} entity
+ * @param {String} name
+ */
+function markAsNotSupported(entity, name) {
+  throw new ReferenceError(
+    "The ${entity} (${name}) is not supported by our driver",
```

Comment:
If you want the error to contain value of entity and name I would recommend using ` instead of " ;)

Reply:
This proves that we need at least a few unit tests that check behaviour of those nonsupported functions when called.